scripts/install-windows-prereqs.ps1

```diff
@@ -645,7 +650,7 @@ function Install-DCAP-Dependencies {
         Throw "Failed to install nuget EnclaveCommonAPI"
     }
 
-    if (($LaunchConfiguration -eq "SGX1FLC") -or (${OS_VERSION} -eq "WinServer2019"))
+    if (($LaunchConfiguration -eq "SGX1FLC") -or (${OS_VERSION} -eq "WinServer2016"))
```

Comment:
Why is this not needed for WS2019?

Reply:
I am running a regression with these changes: https://github.com/BRMcLaren/openenclave/commit/c84e56054c164a1028f2ff40181676075165dcb9 

https://oe-jenkins-tf.westeurope.cloudapp.azure.com/job/CI-CD_Infrastructure/job/OpenEnclave-E2E-Testing/182/

(when it starts)

Reply:
@jhand2 @BRMcLaren 

This shouldn't be changed, and I'm really sorry for not seeing this change in my review.

Marius reverts this change with the PR https://github.com/openenclave/openenclave/pull/2935.